browser/ui/browser_window/internal/browser_window_features.cc

```diff
@@ -80,4 +90,6 @@ void BrowserWindowFeatures::TearDownPreBrowserWindowDestruction() {
     sidebar_controller_->TearDownPreBrowserWindowDestruction();
     playlist_side_panel_coordinator_.reset();
   }
+
+  email_aliases_controller_.reset();
```

Comment:
make sure the order of destruction is correct. I see `brave_vpn_controller_` is constructed last and destroyed first.